src/commissaire/bus/__init__.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, message, code=0, data={}):
         :param data: Additional error data.
         :type data: dict
         """
-        super(RemoteProcedureCallError, self).__init__(message, code, data)
+        super().__init__(message, code, data)
```

Comment:
This is only valid on Python 3, but I think that's fine since we are targeting 3. Side note, if we haven't called out that we require Python 3 we probably should document it.